programming/sed-article.md

```diff
@@ -1016,19 +996,13 @@ All non-blank lines are printed by AutoPrint.
 sed -E -i '/^$/ N; /\n$/ D' my-input.txt
 ```
 
-The first part, `/^$/ N`, appends blank lines to PatSpace.
-Note that PatSpace will be empty after each non-blank line is processed.
-The second part, `/\n$/ D`, deletes the first line in PatSpace
-if PatSpace ends with a newline character.
+The first part, `/^$/ N`, appends blank lines to the pattern space.
+Note that the pattern space will be empty after each non-blank line is processed.
+The second part, `/\n$/ D`, deletes the first line in the pattern space
+if the pattern space ends with a newline character.
 This will be the case every time a blank line is read
 that follows another blank line.
 
-The Unix `cat` command can also be used to do this.
```

Comment:
Why do you feel this isn't worth mentioning? If the goal is to replace all occurrences of consecutive blank lines with a single blank line then it seems using `cat -s` is a better option that using `sed`.

Reply:
To me, it distracts from your main theme because to have to explain by a temporary file is available.